homeassistant/components/tts/__init__.py

```diff
@@ -415,6 +427,18 @@ def async_get_supported_voices(self, language: str) -> list[Voice] | None:
     async def async_internal_added_to_hass(self) -> None:
         """Call when the entity is added to hass."""
         await super().async_internal_added_to_hass()
+        try:
+            _ = self.default_language
+        except AttributeError as err:
+            raise AttributeError(
+                "You need to either set the '_attr_default_language' attribute or override the 'default_language' property"
```

Comment:
Who is "You" here? Assuming this error message is seen by a user, they can't fix it. I'd suggest to rephrase, maybe something like:
"TTS entities must either set the '_attr_default_language' attribute or override the 'default_language' property"